atc/metric/emitter/newrelic.go

```diff
@@ -103,10 +104,21 @@ func (emitter *NewRelicEmitter) emitPayload(logger lager.Logger, payload fullPay
 	resp, err := emitter.client.Do(req)
 
 	if err != nil {
-		logger.Error("failed-to-send-request",
+		logger.Error("new-relic-failed-to-send-request",
```

Comment:
I wonder if rather than prefixing here, we should rely on this being set on the logger session 🤔 

E.g., from the callstack

```
metric/emitter.(*NewRelicEmitter).emitPayload
metric/emitter.(*NewRelicEmitter).Emit
metric.emitLoop()
metric.Initialize
```

the might be a better way of carrying that info around 🤔  

it seems that we just end up adding prefixes to the `influxdb-` to log messages in a recent contribution, but I think that's something we should avoid.

wdyt?